src/encoded/static/components/app.js

```diff
@@ -1803,26 +1801,6 @@ class BodyElement extends React.PureComponent {
         setTimeout(this.throttledScrollHandler, 100, null);
     }
 
-    /**
-     * Is executed after ReactTooltip is hidden e.g. via moving cursor away from an element.
-     * Used to unset lingering style.left and style.top property values which may interfere with placement
-     * of the next visible tooltip.
-     *
-     * @private
-     * @returns {void}
-     */
-    onTooltipAfterHide(){
```

Comment:
This function/mechanism was removed because newest version of React-Tooltip library has fixed or worked around the issue being addressed here wherein the tooltip would previously become invisible yet linger in its placement/position and prevent subsequent tooltips from intersecting that previous placement/position. Was most noticeable if have a row of closely-spaced buttons and hover over them in order which resulted in inconsistent and unexpected tooltip placement.